code/addons/vitest/package.json

```diff
@@ -71,8 +71,8 @@
     "prep": "jiti ../../../scripts/prepare/addon-bundle.ts"
   },
   "dependencies": {
-    "magic-string": "^0.30.10",
-    "typescript": "^5.3.2"
+    "@babel/types": "^7.25.2",
```

Comment:
Do we still need these types if we've moved everything to `csf-tools`?

Reply:
Yeah, we should remove that one